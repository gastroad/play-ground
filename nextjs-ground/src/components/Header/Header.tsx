import { FC } from "react"
import Link from "next/link"

const Header: FC = () => {
    return (
        <>
            <header>
                <nav>
                    <Link href={"todo"} >todo</Link>
                    <br />
                    <Link href={"hello"} >hello</Link>
                </nav>
            </header>

        </>
    )
}

export default Header