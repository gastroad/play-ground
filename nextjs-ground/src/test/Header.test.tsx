import { render, screen } from '@testing-library/react';

import Header from "@src/components/Header"


describe('<Header />', () => {
    it('renders a heading', () => {
        const { container } = render(<Header />);
        const todo = screen.getByText('todo');

        expect(todo).toBeInTheDocument();
        expect(container).toMatchSnapshot();
    });
});