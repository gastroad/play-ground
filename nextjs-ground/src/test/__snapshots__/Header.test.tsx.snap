// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders a heading 1`] = `
<div>
  <header>
    <nav>
      <a
        href="todo"
      >
        todo
      </a>
      <br />
      <a
        href="hello"
      >
        hello
      </a>
    </nav>
  </header>
</div>
`;
